hazelcast-client/src/main/java/com/hazelcast/client/cache/impl/ClientCacheProxy.java

```diff
@@ -403,8 +401,7 @@ protected void updateCacheListenerConfigOnOtherNodes(CacheEntryListenerConfigura
                 final ClientMessage request =
                         CacheListenerRegistrationCodec.encodeRequest(nameWithPrefix, configData, isRegister, address);
                 final ClientInvocation invocation = new ClientInvocation(client, request, address);
-                final Future future = invocation.invoke();
-                futures.add(future);
+                invocation.invoke();
```

Comment:
This actually needs a closer look. The invocation futures were collected, but never used. Is this a bug?